### ISSUE TITLE
If you have issues loading the dataset images and text...

### ISSUE THREAD

**Issue by tonynoce** [kijai/ComfyUI-FluxTrainer#22]
https://github.com/kijai/ComfyUI-FluxTrainer/blob/acf864e850c42f4bbb5eadca9e578285d0db8e7e/examples/flux_lora_train_example01.json#L1617

Im on windows so I moved my dataset to "ComfyUi" folder, in the end it looked like this : "ComfyUI/datasets/newlora/"
Posting this in case another newbie like me doesn't realize.

BTW Thanks Kijai!

**Comment by kijai** [comment]
Yeah for custom nodes the root folder is the ComfyUI root, I was loading my dataset from `datasets` -folder one folder above that.
